scripts/travis/build-all-in-one-image.sh

```diff
@@ -36,13 +37,13 @@ run_integration_test() {
 }
 
 upload_to_docker() {
-  # Only push the docker container to Docker Hub for master/release branch and when dockerhub login is done
-  if [[ ("$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$) && "$DOCKERHUB_LOGIN" == "true" ]]; then
-    echo "upload $1 to Docker Hub"
+  # Only push the docker image to dockerhub/quay.io for master/release branch and when dockerhub or quay.io login is done
+  if [[ ("$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$) && ("$DOCKERHUB_LOGIN" == "true" || "$QUAY_LOGIN" == "true") ]]; then
```

Comment:
I still think we need to push to upload script the login-related checks. If in the future we need to add one more registry, I only want to have to change the upload script. The var checks are not essential here, and we didn't have them before GHA.

Reply:
If I remember correctly, those DOCKERHUB_LOGIN var check wasn't there in Travis days because we used to set DOCKERHUB_TOKEN from Travis UI. After moving to GHA, we set them Github org secrets which aren't accessible in case of PRs that's why we added it. My understanding is if we remove those checks and if someone raises a PR from their fork's master branch, it will try to push images to dockerhub and fail.